src/main/frontend/pipeline-console-view/pipeline-console/main/DataTreeView.tsx

```diff
@@ -27,47 +21,21 @@ export interface StepInfo {
   completePercent: number;
   id: number;
   type: string;
-}
-
-export interface TreeItemData {
-  id: string;
-  name: string;
-  children: TreeItemData[];
+  stageId: string;
 }
 
 interface StageTreeItemProps extends TreeItemProps {
   synthetic?: boolean;
 }
+export interface StepTreeItemProps extends TreeItemProps {
+  stageId: string;
+}
 
 // Tree Item for stages
-const StageTreeItem = withStyles((theme: Theme) =>
-  createStyles({
-    iconContainer: {
-      "& .close": {
-        opacity: 0.3,
-      },
-    },
-    // TODO: Make line show status of block (green = passed, red = failed, etc.)
-    group: {
-      marginLeft: 7,
-      paddingLeft: 18,
-      borderLeft: `1px dashed ${fade(theme.palette.text.primary, 0.4)}`,
-    },
-    label: {
-      fontStyle: (props: StageTreeItemProps) =>
```

Comment:
Maybe I don't understand something but why was this removed? Okay if you don't want synthetic stage labels italicised, but if so they shouldn't probably be italicised in the graph view either...

Reply:
Ah, sorry. I removed the withStyle blocks because I though it was all handled in the scss.